instance/management/commands/migrate_swift_to_s3.py

```diff
@@ -334,6 +334,7 @@ def handle(self, *args, **options):
         """
         Finds all instances and migrates them.
         """
+        self.logger.warning('SWIFT storage is not maintained and will be dropped in a future version.')
```

Comment:
nit: Since this command was specifically designed to migrate from a deprecated system to a non-depricated system I don't think the command itself should be deprecated. 

Although... it will be removed, so maybe?

Reply:
It doesn't work without the django models, so it's deprecated too. :grin: